exporter/exporterhelper/internal/bounded_memory_queue_test.go

```diff
@@ -246,12 +246,3 @@ func (s *consumerState) assertConsumed(expected map[string]bool) {
 	}
 	assert.Equal(s.t, expected, s.snapshot())
 }
-
-func TestZeroSize(t *testing.T) {
```

Comment:
This is covered by not allowing users to configure 0 size, see QueueConfig.Validate()